Author: Tom Jampen <tom@cryptography.ch>
Description:
 Patches check_email_delivery and check_email_delivery_epn to use debian
 specific paths.

Index: pkg-nagios-plugins-contrib/check_email_delivery/src/check_email_delivery
===================================================================
--- pkg-nagios-plugins-contrib.orig/check_email_delivery/src/check_email_delivery
+++ pkg-nagios-plugins-contrib/check_email_delivery/src/check_email_delivery
@@ -63,7 +63,7 @@ my $default_warn = 15;
 my $default_wait = 5;
 my $default_timeout = 60;
 my $time_hires = "";
-my $libexec = "/usr/local/nagios/libexec";
+my $libexec = "/usr/lib/nagios/plugins";
 my $ok;
 $ok = Getopt::Long::GetOptions(
 	"V|version"=>\$show_version,
Index: pkg-nagios-plugins-contrib/check_email_delivery/src/check_email_delivery_epn
===================================================================
--- pkg-nagios-plugins-contrib.orig/check_email_delivery/src/check_email_delivery_epn
+++ pkg-nagios-plugins-contrib/check_email_delivery/src/check_email_delivery_epn
@@ -61,7 +61,7 @@ my $default_warn = 15;
 my $default_wait = 5;
 my $default_timeout = 60;
 my $time_hires = "";
-my $libexec = "/usr/local/nagios/libexec";
+my $libexec = "/usr/lib/nagios/plugins";
 my $ok;
 $ok = Getopt::Long::GetOptions(
 	"V|version"=>\$show_version,
